packages/block-editor/src/layouts/constrained.js

```diff
@@ -125,6 +125,7 @@ export default {
 				) }
 				{ allowJustification && (
 					<ToggleGroupControl
+						__next40pxDefaultSize
```

Comment:
This can be seen for example in the Group block.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/b5693594-089d-4ee4-a390-e4a5de7d4d9f" alt="Constrained layout controls, before" width="275">|<img src="https://github.com/user-attachments/assets/89622941-f3bf-42b6-912a-00d73393e30f" alt="Constrained layout controls, after" width="275">|